packages/inferno-animation/src/animationCoordinator.ts

```diff
@@ -0,0 +1,75 @@
+import { forceReflow } from './utils';
+
+export enum AnimationPhase {
+  INITIALIZE,
+  MEASURE,
+  SET_START_STATE,
+  ACTIVATE_TRANSITIONS,
+  REGISTER_LISTENERS,
+  ACTIVATE_ANIMATION
+}
+
+let _animationQueue: Function[] = [];
+let _animationActivationQueue: Function[] = [];
+let _nextAnimationFrame;
+let _nextActivateAnimationFrame;
```

Comment:
we can do one micro-optimization by keeping this variable as long integer value and using 0 as a non-set value.

This way Javascript engine can make it an integer, if we assign it undefined it may de-opt

https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame#return_value

Reply:
WILL FIX